resources/language/resource.language.de_de/strings.po

```diff
@@ -692,7 +692,7 @@ msgstr "Aktiviere Workaround für STRM-Wiedergabe-Fortsetzung"
 
 msgctxt "#30177"
 msgid "Ask to resume the video"
-msgstr "Frage zum Fortsetzen des Videos"
+msgstr "Fragen ob Video fortgesetzt werden soll"
```

Comment:
There is a comma missing after "Fragen".

Reply:
Thanks. fixed!